galaxy_importer/main.py

```diff
@@ -61,8 +62,12 @@ def call_importer(file):
     """
     try:
         data = collection.import_collection(file)
+    except ImporterError as e:
+        logging.info('The import failed, the following '
+                     f'ImporterError was raised: {e!r}')
```

Comment:
I think it's better to print a human readable error message or a technical traceback depending on the auditory. This message tries to be both at the same time, because of `repr()` function used and internal exception type written.

For python Exception class `repr()` function returns something like `"Exception('foo')"`, not the exception message string. To get message string you should use `str()`